docs/reference/search/profile.asciidoc

```diff
@@ -784,46 +784,52 @@ This yields the following aggregation profile output:
           {
             "type" : "LongTermsAggregator",
             "description" : "my_scoped_agg",
-            "time_in_nanos" : 195386,
+            "time_in_nanos" : 195421,
             "breakdown" : {
               "reduce" : 0,
-              "build_aggregation" : 81171,
+              "build_aggregation" : 80927,
               "build_aggregation_count" : 1,
-              "initialize" : 22753,
+              "initialize" : 3619,
               "initialize_count" : 1,
               "reduce_count" : 0,
-              "collect" : 91456,
-              "collect_count" : 4
+              "collect" : 109778,
+              "collect_count" : 4,
+              "run_deferred_collections" : 1090,
+              "run_deferred_collections_count" : 1
```

Comment:
I had to add these lines so I took the opportunity to update the values above with results from my test.